test/utils/CachedItem.ts

```diff
@@ -0,0 +1,38 @@
+import CachedItem from '../../src/utils/CachedItem';
+import { Organization } from '../../src/apiClients/AirTableApiClient';
+import sinon from 'sinon';
+import test from 'ava';
+
+const org: Organization = {
+  name: 'Chrysalis Enterprises',
+  riverSection: 'Upper',
+  activity: 'Needs Volunteers',
+  description:
+    'Chrysalis is a nonprofit organization dedicated to creating a pathway to self-sufficiency for homeless and low-income individuals by providing the res...',
+  email: 'fakeemail@chrysalis.org',
+  phoneNumber: '(123) 456-7890',
+  url: 'https://changelives.org/',
+  interestCategories: ['Water Organizations'],
+};
+
+const cache: CachedItem<Organization> = new CachedItem<Organization>(3000);
```

Comment:
You should use a separate `CacheItem` instance for each test so that they're not interdependent.